### DISCUSSION TITLE
.drawio file render support like .ipynb

### DISCUSSION THREAD

**Discussion by tallguyjenks** [community/community#4344] [Other Feature Feedback, Questions, & Ideas] [answered]
Having support for draw.io files rendering on github would be amazing. That and/or native mermaid support documented diagrams. 

[draw.io](https://app.diagrams.net/)
[mermaid](https://github.com/mermaid-js/mermaid)

**Comment by davidstaheli** [comment, marked as answer]
Hi @tallguyjenks. Mermaid support was [released today](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/). GitHub wants to support draw.io too, but it hasn't been scheduled yet. We're in the middle of investigation that will determine how soon we can support it.

**Reply by tallguyjenks** [reply]
That's awesome!
